go/vt/vttablet/tabletserver/schema/historian.go

```diff
@@ -20,49 +20,55 @@ import (
 	"context"
 	"sort"
 	"sync"
+	"time"
+
+	"vitess.io/vitess/go/vt/sidecardb"
 
 	"vitess.io/vitess/go/mysql"
 	"vitess.io/vitess/go/sqltypes"
 	"vitess.io/vitess/go/vt/log"
 	binlogdatapb "vitess.io/vitess/go/vt/proto/binlogdata"
-	"vitess.io/vitess/go/vt/sidecardb"
 	"vitess.io/vitess/go/vt/vtgate/evalengine"
 	"vitess.io/vitess/go/vt/vttablet/tabletserver/connpool"
 	"vitess.io/vitess/go/vt/vttablet/tabletserver/tabletenv"
 
 	"vitess.io/vitess/go/vt/sqlparser"
 )
 
-const getSchemaVersions = "select id, pos, ddl, time_updated, schemax from %s.schema_version where id > %d order by id asc"
+const getInitialSchemaVersions = "select id, pos, ddl, time_updated, schemax from %s.schema_version where from_unixtime(time_updated) < %d order by id asc"
```

Comment:
Don't we want the records time_updated to be _greater_ than the value we pass in? We pass in e.g. now()-1 hour and we only want things that are newer than that, no?

We should test this behavior as well. 

We should also call it out in the PR description.